test-it/karma/fixtures/publicapi/MultipleTypes.via

```diff
@@ -0,0 +1,90 @@
+// Autogenerated Vireo assembly file
```

Comment:
My question was not about that. I was wondering whether you were planning to checkin GVI tests along with the other existing GVI tests after this goes in on the Vireo side?